prime-dotnet-webapi/Models/Statuses/StatusType.cs

```diff
@@ -6,6 +6,12 @@ public enum StatusType
         UnderReview = 2,
         RequiresToa = 3,
         Locked = 4,
-        Declined = 5,
+        Declined = 5
+    }
+
+    public enum AdminStatusType
+    {
+        Enabled = 1,
```

Comment:
Wondering if this should be 7 instead since in the Angular code, `AccessStatusEnum` is used and 1 in that set is `UNDER_REVIEW` 🤔

Reply:
In order to avoid regenerating the migration, I update the createAdmin() function to use AdminStatusType.Enabled instead. It doesn't matter what the actual value is.

Reply:
Thanks for your patience @bergomi02, I'm done reviewing code.  Note that I committed some minor changes for stronger data typing but I think `AdminStatusType.Enabled` should still have a numeric value outside of `StatusType` range.

Reply:
Never mind